src/components/grid-aware/ArticleSpotlightCard/ArticleSpotlightCard.stories.js

```diff
@@ -26,10 +26,14 @@ const Template = ({
 
 export const Default = Template.bind({});
 Default.args = {
-  eyebrowText: "Partnership Spotlight",
-  title: "Digital Equity Team of San Francisco",
+  eyebrowText: "ShelterConnect Spotlight",
+  title: "Mission Hotel",
```

Comment:
There's not really a need to change the Storybook story. This component will be used in multiple places, and we will periodically update the cards as we have new things to highlight, so this Storybook story doesn't have always be updated with it.

This is partially why I wanted to separate the images used by the Storybook stories from the images used in the actual pages.